### PR TITLE
updated numpy version

### PR DESCRIPTION
**Pull request by AhtishamShahid**
Updated Numpy version to 1.16.5

### CODE REVIEW

File: apt-packages.txt [hunk 1]
```diff
@@ -2,9 +2,9 @@ python-pip
 gcc
 g++
 gfortran
-libblas3gf
+
 libblas-dev
-liblapack3gf
```
Comment (awaisdar001) [inline apt-packages.txt]: please add a comment why you are removing these.

**Comment by awaisdar001** [comment]
@AhtishamShahid please rebase your branch to have only one commit.

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+dist: trusty
```
Comment (awaisdar001) [inline .travis.yml]: Please share some context here why you required to specify the distribution?

Comment (AhtishamShahid) [reply .travis.yml]: The apt packages `libblas3gf`, `liblapack3gf`  cannot be installed on ubuntu `version >14` but by default, Travis uses ubuntu `16`. so I have to pin ubuntu `14` code name `trusty`.

For further details checkout this [thread](https://askubuntu.com/questions/989778/e-unable-to-locate-package-libblas3gf)

**Pull request merged**